src/components/stateScreen/index.js

```diff
@@ -22,6 +22,7 @@ export default class StateScreen extends BaseComponent {
      * The image source that's showing at the top. use an image that was required locally
      */
     imageSource: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
+    source: PropTypes.oneOfType([PropTypes.object, PropTypes.number]), //TODO: Remove after imageSource deprecation
```

Comment:
(Same here) - not sure about the TODO, I think in this case imageSource will be removed and source will stay

Reply:
Same here

Reply:
Ok